packages/react/src/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./Breadcrumb.css";
+import { useBreadcrumbItems } from "./useBreadcrumbItems";
+
+export type BreadcrumbProps = BoxProps<
+  "nav",
+  {
+    children: React.ReactNode;
+    maxItems?: number;
+    separator?: React.ReactNode;
+  }
+>;
+
+export const Breadcrumb = forwardRef<HTMLElement, BreadcrumbProps>(
+  (
+    { children, className, maxItems = Infinity, separator = ">", ...props },
```

Comment:
Do we really need to set Infinity rather than a number as default?

Reply:
This needs to be discussed together